fetch

```diff
@@ -1389,6 +1389,11 @@ getbattery () {
             fi
         ;;
 
+        "BSD")
```

Comment:
The reason `OpenBSD` has a seperate line in `case "$(uname)"` part of the script
is that the output of `uname` om OpenBSD is entirely different to all other BSDs for 
some reason